code/game/turfs/simulated/floor.dm

```diff
@@ -54,8 +54,8 @@
 		icon_regular_floor = "floor"
 	else
 		icon_regular_floor = icon_state
-	if(mapload && prob(33))
-		MakeDirty()
+	//if(mapload && prob(33)) disabled for now requires a rework at dirtystation.dm
```

Comment:
//DeepSpace13 - Disabled for now, requires a rework at dirtystation.dm

Reply:
Understood

Reply:
//Deep Space 13 - Comment